### PR TITLE
Cpp kv pipeline

### PR DESCRIPTION
**Pull request by gearama**
Add cpp to KV secrets perf tests, also added support for KV keys and certificates perf tests

### CODE REVIEW

File: tools/perf-automation/Azure.Sdk.Tools.PerfAutomation/tests.yml [hunk 1]
```diff
@@ -606,3 +616,41 @@ Services:
...
+      - azure-security-keyvault-certificates-cpp: source
+        azure-core-cpp: source
+
+  Tests:
+  - Test: get-certificate
+    Arguments:
+    - --warmup 0 --duration 1
+    - --parallel 64
+    TestNames:
+      Cpp: GetCertificate
```
Comment (mikeharder) [inline tools/perf-automation/Azure.Sdk.Tools.PerfAutomation/tests.yml:656]: NIT: Missing newline at end of file

**Review by mikeharder — APPROVED** [review]
I'm fine with these additions, but note the KeyVault service has throttles well below our client performance limits, so tests need to use a test proxy to measure the maximum client throughput.  We are planning to add support for test-proxy to our perf pipelines in the future, but it's not ready yet.
